### DISCUSSION TITLE
Add Rocky Linux to the Official ASCII art!

### DISCUSSION THREAD

**Discussion by Kha1r0** [fastfetch-cli/fastfetch#893] [Ideas]
basically says it in the title

**Comment by CarterLi** [comment]
You should fill the logo request template in Issues section.

**Reply by Kha1r0** [reply]
Thanks!

**Comment by Kha1r0** [comment]
https://github.com/fastfetch-cli/fastfetch/issues/895

**Reply by CarterLi** [reply]
You selected the wrong template

**Reply by Kha1r0** [reply]
oh my god i'm an idiot
